File: Components/WinForms/CS/SchedulerMainDemo/CarsData/CarScheduling.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DevExpress.XtraScheduler.Demos {
    public class CarScheduling {
        [Key]
        public Int64 Id { get; set; }
        public bool AllDay { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public string Description { get; set; }
        public int Status { get; set; }
        public int Label { get; set; }
        public int EventType { get; set; }
        public string Location { get; set; }
        public string Subject { get; set; }
        public string RecurrenceInfo { get; set; }
        public string ReminderInfo { get; set; }
        public Int32? CarId { get; set; }
        public double Price { get; set; }
    }
}

File: Components/WinForms/VB/SchedulerMainDemo/CarsData/CarScheduling.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace DevExpress.XtraScheduler.Demos
	Public Class CarScheduling
		<Key> _
		Public Property Id() As Int64
		Public Property AllDay() As Boolean
		Public Property StartTime() As DateTime
		Public Property EndTime() As DateTime
		Public Property Description() As String
		Public Property Status() As Integer
		Public Property Label() As Integer
		Public Property EventType() As Integer
		Public Property Location() As String
		Public Property Subject() As String
		Public Property RecurrenceInfo() As String
		Public Property ReminderInfo() As String
		Private privateCarId? As Int32
		Public Property CarId() As Int32?
			Get
				Return privateCarId
			End Get
			Set(ByVal value? As Int32)
				privateCarId = value
			End Set
		End Property
		Public Property Price() As Double
	End Class
End Namespace
